### PR TITLE
feat: Adds 3 PLA materials from Layer One

### PR DESCRIPTION
**Pull request by Daniel-Kurth**
Adds files for Layer Ones PLA materials in 3 colors.

### CODE REVIEW

File: layer_one_white_pla.xml.fdm_material [hunk 1]
```diff
@@ -0,0 +1,47 @@
+<?xml version='1.0' encoding='utf-8'?>
+<fdmmaterial version="1.3" xmlns="http://www.ultimaker.com/material" >
+    <!-- Template at: https://github.com/Ultimaker/fdm_materials/blob/master/ultimaker_pva.xml.fdm_material -->
```
Comment (nallath) [inline layer_one_white_pla.xml.fdm_material:3]: This isn't PVA

File: layer_one_white_pla.xml.fdm_material [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    <name>
+      <brand>Layer One</brand>
+      <material>PLA</material>
+      <color>White</color>
+      <label>Standard White</label>
+    </name>
+
+     <version>1</version>
+     <GUID>af740bcf-6e7e-457d-a3bb-534699b3378a</GUID>
+     <color_code>#ffffff</color_code><!-- Unknown if this is correct -->
```
Comment (nallath) [inline layer_one_white_pla.xml.fdm_material:14]: Why do you have doubts about this?

**Comment by Daniel-Kurth** [comment]
I removed all the unneeded comments and also adjusted the one of the color values. Sorry for not having caught the comments and adding so many commits in between.

**Review by Ghostkeeper — APPROVED** [review]
Aside from the aforementioned comments this looks fine. We'll give this a test in the application and merge it if it works. I'll remove the irrelevant PVA comments myself.

**Pull request merged**